osu.Game/osu.Game.csproj

```diff
@@ -18,7 +18,7 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="2.2.1" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite.Core" Version="2.2.1" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
-    <PackageReference Include="ppy.osu.Framework" Version="2019.117.0" />
+    <PackageReference Include="ppy.osu.Framework" Version="0.0.8033" />
```

Comment:
Version from local build?

Reply:
from appveyor build (see osu-framework wiki page for instructions)

Reply:
Then build scipt missing adding appveyor source(in case of CI build)